dashboard/controller.py

```diff
@@ -88,6 +99,45 @@ def show_builds_history(page=1):
                            buildsets_log_url=buildset_log_url)
 
 
+@build_enqueue.route('/control')
+def control_panel():
+    if session and session['user'] and session['user'].is_admin():
+        return render_template('control_panel.html',
+                               start_pipelines=config['build_enqueue'][
+                                   'pipelines'])
+    abort(requests.codes.unauthorized)
+
+
+@build_enqueue.route('/start', methods=['POST'])
+def start_buildset():
```

Comment:
`start_buildset` and `stop_buildset` looks exactly the same. Can we merge it under `route('/control/manage')` instead of high level `/start` `/stop`?